RELEASE.md

```diff
@@ -1,9 +1,15 @@
 To release a new version of the widgets on PyPI:
 
-1. Update \_version.py
+1. Update \_version.py (set release version, remove 'dev')
 2. `git add` and `git commit`
 3. `python setup.py sdist upload` 
 4. `python setup.py bdist_wheel upload`
```

Comment:
Me too, just that it is different from the wheels tutorial, hence the question. If it worked, you answered my question :)